Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,32 @@
 import React from 'react';
+import HeaderContent from './HeaderContent';
 import './Header.css';
+import '../../App.css';
+import moment from 'moment'
```

Comment:
You need to install this package from the right folder in order to be able to use it.